src/pages/pages.js

```diff
@@ -0,0 +1,11 @@
+import Explore from "./Explore/Explore";
+import History from "./History/History";
+import Login from "./Authentication/Login"
+import SignUp from "./Authentication/SignUp"
+import Liked from "./Liked/Liked";
+import MockApi from "./Mockman/MockApi";
+import PageNotFound from "./PageNotFound/PageNotFound";
+import Playlist from "./Playlist/Playlist";
+import WatchLater from "./WatchLater/WatchLater";
+
+export {Explore, History, Login, SignUp, Liked, MockApi,PageNotFound, Playlist, WatchLater}
```

Comment:
Named exports allow us to export directly. So we don't have to import it first and then export it like how you've done. 
```javascript
export { Login } from "./Authentication/Login"
```
You can play with it later.

Reply:
Oh! I wasn't aware of this. Thank you!